addons/automatic_updates.sh

```diff
@@ -18,29 +21,30 @@ debug_mode
 # Check if root
 root_check
 
-msg_box "This option will update your server every week on Saturdays at $AUT_UPDATES_TIME:00.
-The update will run the built in script '$SCRIPTS/update.sh' which will update both the server packages and Nextcloud itself.
-
-You can read more about it here: https://www.techandme.se/nextcloud-update-is-now-fully-automated/
-Please keep in mind that automatic updates might fail hence it's \
-important to have a proper backup in place if you plan to run this option.
-
-You can disable the automatic updates by entering the crontab file like this:
-'sudo crontab -e -u root'
-Then just put a hash (#) in front of the row that you want to disable.
-
-In the next step you will be able to choose to proceed or exit." "$SUBTITLE"
+# Check if automatic updates are already installed
+if ! crontab -u root -l | grep -q "$SCRIPTS/update.sh"
+then
+    # Ask for installing
+    install_popup "$SCRIPT_NAME"
+else
+    # Ask for removal or reinstallation
+    reinstall_remove_menu "$SCRIPT_NAME"
+    # Removal
+    crontab -u root -l | grep -v "$SCRIPTS/update.sh"  | crontab -u root -
+    sed -i "s|/sbin/shutdown -r +1||g" "$SCRIPTS"/update.sh
```

Comment:
I think we use `reboot` in an earlier version as well. Please check history.

Reply:
hm... I think I am just deleting this line... done

Reply:
No, we don't want to reboot if it's uninstalled... Don't make it easy, make it stable. :D

Reply:
done